packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_licensed_items_purchases.py

```diff
@@ -26,14 +26,30 @@
     ),
     sa.Column(
         "licensed_item_id",
-        sa.BigInteger,
+        UUID(as_uuid=True),
```

Comment:
Q: since you changed the type here, I have two questions:
- do you have other items that already used this id in the system and stored previous versions?
- I guess you are replacing the old id with a random uuid?

Reply:
This is totally new table created in the previous PR not used yet anywhere. I just wrongly defined type in the previous PR.